github_flow.md

```diff
@@ -0,0 +1,48 @@
+# Github flow
+
+## We have agreed, that we will work using Github flow. Simple instructions can be found below.
+
+In short words, we are forking repository from Code for Poznan organisation to our repository, push the changes there and create pull requests from our branches (eg. w1stler/sample_branch) to codeforpoznan/master. 
+
+## Link for reference: https://guides.github.com/introduction/flow/
+
+Example config for proper Github flow.
+
+```
+[core]
```

Comment:
I have never (or at least don't remember) changed this file (and You have not even give it name or location).

I would suggest section named `How to setup Your local repository` which will contains:
1. Fork from Code for Poznan
1. Clone Your (forked) repository to Your local computer.
1. Add upstream remote (simply execute command `git remote add upstream it@github.com:CodeForPoznan/project_name.git`

And that should be all